etc/docker/node-init.sh

```diff
@@ -63,7 +63,7 @@ if [[ "$parsedVersion" -lt "37" ]] ; then
 fi
 nsenter -t 1 -m -u -n -i mkdir -p /opt/cni/bin && \
 nsenter -t 1 -m -u -n -i mkdir -p /etc/cni/net.d && \
-nsenter -t 1 -m -u -n -i ln -snf /var/mizar/mizar/cni.py /opt/cni/bin/mizarcni && \
+nsenter -t 1 -m -u -n -i ln -snf /var/mizar/build/bin/mizarcni /opt/cni/bin/mizarcni && \
```

Comment:
Same here. 

Why do we need both node-init.sh and node-init.cni.sh?

Reply:
May be historic reason? @phudtran do you know? 

The both files are there so I updated both. I see the both files are being called so I think it's not a good idea to remove any of one.

Reply:
Removed.